lib/ansiblelint/color.py

```diff
@@ -0,0 +1,18 @@
+"""Console coloring support."""
+from enum import Enum
+
+
+class Color(Enum):
+    """Color styles."""
+
+    reset = "0"
+    error_code = "1;31"  # bright red
+    error_title = "0;31"  # red
+    filename = "0;34"  # blue
+    linenumber = "0;36"  # cyan
+    line = "0;35"  # purple
+
+
+def stringc(text: str, color: Color):
```

Comment:
Since it's a function, it should be a verb:
```suggestion
def colorize(text: str, color: Color):
```